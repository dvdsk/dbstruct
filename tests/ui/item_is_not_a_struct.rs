use dbstruct::dbstruct;

#[dbstruct]
fn F() {}

fn main() {}
