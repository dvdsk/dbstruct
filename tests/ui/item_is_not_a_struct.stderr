error: expected `struct`
 --> tests/ui/item_is_not_a_struct.rs:4:1
  |
4 | fn F() {}
  | ^^
